app/src/main/res/values/strings.xml

```diff
@@ -33,6 +33,13 @@
     <string name="edit_menu_item">Edit</string>
     <string name="delete_menu_item">Delete</string>
 
+    <string name="activating_bluetooth_went_wrong">Bluetooth could not be activated</string>
+    <string name="activating_bluetooth_went_wrong_explanation">Bluetooth needs to be activated to share public keys over Bluetooth.
+    Please try again or use another method.</string>
```

Comment:
Not sure, "contacts" just feels a bit more intuitive but I guess you're right that this is about keys 🤔